### ISSUE TITLE
Tooltip fade

### ISSUE THREAD

**Issue by danejasho** [project-classic/ClassicCodex#49]
The tooltip for the mouse cursor (mobs, npc's, etc) insta-fades when moused off. Can this be changed to behave identically to the classic client, with a gradual fade.

**Comment by SwimmingTiger** [comment]
I think we are using the standard World of Warcraft [GameTooltip](https://wowwiki.fandom.com/wiki/UIOBJECT_GameTooltip) component.

So this should be the default behavior of World of Warcraft client. There are currently no CVar to modify the behavior.

But some addons seem to be able to fade out of `GameTooltip`.
Not tested whether it is compatible with classic.
https://curseforge.com/wow/addons/tip-tac
```
Configurable fading options, both a prefade and fadeout time can be set. They can also be set to zero if you prefer the tips to just hide instantly.
```

**Comment by pgarneau** [comment]
We aren't using the standard WoW tooltip, but I used to before.

When there was the fade mechanic, the tooltip wouldn't update until it completely disappeared and I couldn't find the "event" that triggered a "redraw" of the tooltip. What would happen is that if you hovered over a mob, would show "some item 6/10" and if u changed cursor location, it would start to fade and if you put your cursor back on the mob, it wouldn't have the "some item 6/10". Couldn't get it to work so I made the tooltip disappear / appear instantly when hovering over things
